packages/fxa-content-server/app/scripts/views/sms_send.js

```diff
@@ -31,11 +32,38 @@ class SmsSendView extends FormView {
   mustAuth = true;
   template = Template;
 
+  initialize() {
+    this.events = assign(this.events, {
+      'input .phone-number': '_validatePhoneNumber',
+    });
+  }
+
+  afterRender() {
+    // Make this a jQuery object so we can use the validation helpers
+    this.phoneField = this.el.querySelector('.phone-number');
+    this.$phoneField = this.$(this.phoneField);
+    this.submitButton = this.el.querySelector('.sms-send');
+
+    this._validatePhoneNumber();
+  }
+
   getAccount() {
     // TODO - remove the `|| ...` when done with development
     return this.model.get('account') || this.user.getSignedInAccount();
   }
 
+  _validatePhoneNumber() {
+    try {
+      if (!this.phoneField.disableValidation) {
```

Comment:
Can we put `// for testing only` or something of the sort if that's all we're using this for?